Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -225,7 +228,7 @@ def get_machine_guid(client: Client, machine_name: str) -> str:
 
 
 def is_probe_connected_command(client: Client, args: dict, is_remediation_commmand: bool = False) -> Any:
-    machine = args.get('machine')
+    machine = str(args.get('machine'))
```

Comment:
No need to convert to a string as the demisto.args() returning all the arguments as str already. Did you encounter any issue with it?

Reply:
Yes, we were getting a MyPy error if we didn't added the typecast. Please let me know if you want us to remove it.